src/Process/Builder/InitialTestRunProcessBuilder.php

```diff
@@ -36,36 +36,28 @@
 namespace Infection\Process\Builder;
 
 use Infection\Console\Util\PhpProcess;
-use Infection\Mutant\MutantInterface;
-use Infection\Process\MutantProcess;
 use Infection\TestFramework\AbstractTestFrameworkAdapter;
 use Symfony\Component\Process\Process;
 
 /**
  * @internal
  */
-class ProcessBuilder
+class InitialTestRunProcessBuilder
 {
     /**
      * @var AbstractTestFrameworkAdapter
      */
     private $testFrameworkAdapter;
 
-    /**
-     * @var int
-     */
-    private $timeout;
-
-    public function __construct(AbstractTestFrameworkAdapter $testFrameworkAdapter, int $timeout)
+    public function __construct(AbstractTestFrameworkAdapter $testFrameworkAdapter)
     {
         $this->testFrameworkAdapter = $testFrameworkAdapter;
-        $this->timeout = $timeout;
     }
 
     /**
      * Creates process with enabled debugger as test framework is going to use in the code coverage.
      */
-    public function getProcessForInitialTestRun(
+    public function createProcessForInitialTestRun(
```

Comment:
`createProcess()`? the name is a bit redundant with the class name...:

```php
$initialTestRunProcessBuilder->createProcessForInitialTestRun();

// IMO this is clear enough:
$initialTestRunProcessBuilder->createProcess();
```

Not feeling strongly about it though